AGENT_CHANGELOG.md

```diff
@@ -114,7 +119,7 @@
 * FoundationDB [1.2.0](https://github.com/DataDog/integrations-core/blob/master/foundationdb/CHANGELOG.md)
 * Gitlab Runner [3.3.0](https://github.com/DataDog/integrations-core/blob/master/gitlab_runner/CHANGELOG.md)
 * Gitlab [5.3.0](https://github.com/DataDog/integrations-core/blob/master/gitlab/CHANGELOG.md)
-* Go-Expvar [2.2.0](https://github.com/DataDog/integrations-core/blob/master/go_expvar/CHANGELOG.md)
```

Comment:
?

Reply:
I think that was due to this change:
[#13789https://github.com/DataDog/integrations-core/pull/13789](https://github.com/DataDog/integrations-core/pull/13789)